### ISSUE TITLE
use ant-design-vue2.x Ignore build dist css incomplete

### ISSUE THREAD

**Issue by nshusr** [windicss/vite-plugin-windicss#130]
**Describe the bug**

Introducing ant-design-vue 2.x and using vite build will cause antd.css and windi more css dist packaging to be ignored

packaged dist css incomplete

main.ts
```
...
import 'ant-design-vue/dist/antd.css'
import 'virtual:windi.css'
import 'virtual:windi-devtools'
...
```
![image](https://user-images.githubusercontent.com/45023042/113570969-ff008280-9647-11eb-8bc2-060512f19bc1.png)
**Minimal Reproductions**

https://github.com/nshusr/antd-windi-bug-repo

**Versions**
    "vite-plugin-windicss": "^0.12.2",
    "vite": "^2.1.5",
    "vue": "3.0.11",

**Comment by antfu** [comment]
Fixed in 0.12.5

**Comment by nshusr** [comment]
@antfu 
v0.14.6 This problem reappears in this version
![image](https://user-images.githubusercontent.com/45023042/115126730-fc1c6f00-a003-11eb-9384-aa92c30e7ed9.png)
